programs/voter-stake-registry/src/lib.rs

```diff
@@ -15,7 +16,7 @@ mod instructions;
 pub mod voter;
 
 // The program address.
-declare_id!("9XZ7Ku7FYGVk3veKba6BRKTFXoYJyh4b4ZHC6MfaTUE8");
+declare_id!("C62KFt3Ujv4GmgaNRpTzyvkERVW2jPSje7cGaw4L5hAx");
```

Comment:
we have the same id defined in program-states, do we need both?

Reply:
`program-states` should contain id because it's anchor-based, however, there's indeed no sense in keeping them identical.

Reply:
We do. Otherwise, the anchor will consider imported structures as they were imported from another contract.